decidim-forms/lib/decidim/exporters/form_pdf.rb

```diff
@@ -10,24 +8,141 @@ module Exporters
+        def answers
+          siblings.map { |answer| QuestionnaireAnswerPresenter.new(answer:) }
+        end
       end
 
-      def layout
-        "decidim/forms/admin/questionnaires/questionnaire_answers"
+      protected
+
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.question.title')
+      def add_data!
+        composer.text(translated_attribute(questionnaire.title), style: :h1)
+        composer.text(decidim_sanitize_translated(questionnaire.description), style: :description)
+        composer.text(I18n.t("question.title", scope:, count: collection.count), style: :section_title)
+
+        local_collection = collection.map { |answer| ParticipantPresenter.new(participant: answer.first) }
+
+        local_collection.each_with_index do |record, index|
+          add_response_box(record, index)
+        end
       end
 
-      def locals
+      def scope = "decidim.forms.admin.questionnaires.export"
+
+      def questionnaire = collection.first.first.questionnaire
+
+      def styles
         {
-          questionnaire: collection.first.first.questionnaire,
-          collection: collection.map { |answer| Decidim::Forms::Admin::QuestionnaireParticipantPresenter.new(participant: answer.first) }
+          h1: { font: composer.document.fonts.add(font_family, variant: :bold), font_size: 18, margin: [0, 0, 10, 0] },
+          th: { font: composer.document.fonts.add(font_family, variant: :bold), font_size: 10, margin: [0, 0, 10, 0] },
+          td: { font: composer.document.fonts.add(font_family), font_size: 10, margin: [0, 0, 10, 0] },
+          description: { font: composer.document.fonts.add(font_family), margin: [0, 0, 10, 0], font_size: 10 },
+          section_title: { font: composer.document.fonts.add(font_family, variant: :bold), margin: [15, 0, 15, 0], font_size: 16 },
+          question_title: { font: composer.document.fonts.add(font_family, variant: :bold), margin: [10, 0, 10, 0], font_size: 12 },
+          question_answer: { font: composer.document.fonts.add(font_family), margin: [10, 0, 10, 0], font_size: 10 },
+          file_info: { font: composer.document.fonts.add(font_family), margin: [10, 0, 10, 0], font_size: 8 },
+          link: { font: composer.document.fonts.add(font_family), margin: [10, 0, 10, 0], font_size: 10, underline: true }
         }
       end
+
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.session_token')
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.user_status')
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.ip_hash')
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.completion')
+      # i18n-tasks-use t('decidim.forms.admin.questionnaires.export.created_at')
+      def header
+        [
+          layout.text(I18n.t("session_token", scope:), style: :th),
+          layout.text(I18n.t("user_status", scope:), style: :th),
```

Comment:
Can you reuse the i18n strings that were already there t("user_status", scope: (like `"decidim.forms.user_answers_serializer")`)? I think we should try to minimize the number of new strings added, specially on issues like this where we should try to keep it really similar to what was already there.

Reply:
fixed in [1b8a59c](https://github.com/decidim/decidim/pull/13617/commits/1b8a59c8c929ce9756c3e9906e53a9ef54132e10)